product-scenarios/scenarios-commons/src/main/java/org/wso2/carbon/esb/scenario/test/common/ScenarioTestBase.java

```diff
@@ -276,17 +282,46 @@ public Boolean call() throws Exception {
         };
     }
 
-    protected String getStringFromDocument(Document doc) throws TransformerException {
-        DOMSource domSource = new DOMSource(doc);
-        StringWriter writer = new StringWriter();
-        StreamResult result = new StreamResult(writer);
-        TransformerFactory tf = TransformerFactory.newInstance();
-        Transformer transformer = tf.newTransformer();
-        transformer.transform(domSource, result);
-        return writer.toString();
+    protected List<Object[]> getRequestResponseList(String testCase) throws Exception {
```

Comment:
Shall we add method comment?

Reply:
This method was updated and method comment was added in PR [https://github.com/wso2/product-ei/pull/2937](https://github.com/wso2/product-ei/pull/2937)